qa/tasks/vstart_runner.py

```diff
@@ -465,7 +466,15 @@ def run(self, **kwargs):
     # the path to binary file.
     def _do_run(self, args, check_status=True, wait=True, stdout=None,
                 stderr=None, cwd=None, stdin=None, logger=None, label=None,
-                env=None, timeout=None, omit_sudo=True, shell=True, quiet=False):
+                env=None, timeout=None, omit_sudo=True, shell=True,
+                quiet=False, terminate=False):
+        '''
+        :param termimate: set this to True when the command is expected to be
```

Comment:
Why is this required?

Reply:
Same: we talked about this change last time. Anyways, it has been removed now.